eng/pipelines/corefx-base.yml

```diff
@@ -186,26 +186,50 @@ jobs:
                   artifactType: container
                 condition: and(succeeded(), eq(variables['_publishTests'], 'true'))
 
-            - template: /eng/pipelines/helix.yml
-              parameters:
-                # send tests to helix only on public builds, official scheduled builds or manual official builds.
-                condition: eq(${{ parameters.isOfficialBuild }}, False)
-                targetOS: ${{ parameters.targetOS }}
-                archGroup: $(_architecture)
-                configuration: $(_BuildConfig)
-                helixQueues: $(_helixQueues)
-                msbuildScript: $(_msbuildCommand)
-                framework: $(_framework)
-                testScope: ${{ coalesce(parameters.testScope, 'innerloop') }} # if parameters.testScope is empty use 'innerloop'
-
-                ${{ if eq(parameters.isOfficialBuild, 'true') }}:
-                  officialBuildId: $(Build.BuildNumber)
-                  helixToken: $(HelixApiAccessToken)
-
-                ${{ if eq(parameters.isOfficialBuild, 'false') }}:
-                  # TODO: SET Creator to the PR owner whenever Azure DevOps supports a good way to retrieve it.
-                  creator: dotnet-bot
-                  helixToken: ''
+            - ${{ if contains(variables['Build.DefinitionName'], 'runtime') }}: # TODO: (Consolidation) remove.
```

Comment:
I had to duplicate this block unfortunately because of the way this yaml file is expanded via: `job.yml`... since all that happens at compile time before the variables template is imported, I can't use it here 😢